include/networkit/centrality/TopHarmonicCloseness.hpp

```diff
@@ -105,8 +117,8 @@ class TopHarmonicCloseness final : public Algorithm {
         }
     }
 
-    tlx::d_ary_addressable_int_heap<node, 2, Aux::GreaterInVector<double>> prioQ;
-    tlx::d_ary_addressable_int_heap<node, 2, Aux::LessInVector<double>> topKNodesPQ;
+    std::unique_ptr<tlx::d_ary_addressable_int_heap<node, 2, Aux::LessInVector<double>>>
+        topKNodesPQPtr;
```

Comment:
Why did you change this to a `unique_ptr`?

Reply:
Good point. I had issues with non-emptied `tlx::d_ary_addressable_int_heap` during refactoring and was unsure, whether the datastructure really gets reset between runs. This is not anymore needed - will revert it.